CMakeLists.txt

```diff
@@ -1142,16 +1142,12 @@ if (APPLE)
         MACOSX_BUNDLE_INFO_PLIST "${CMAKE_CURRENT_SOURCE_DIR}/packaging/macos/bundle.plist.in"
     )
   endif()
-else()
+endif()
+
+if(WIN32)
+  set(CMAKE_INSTALL_SYSTEM_RUNTIME_DESTINATION "${MIXXX_INSTALL_BINDIR}")
+  set(CMAKE_INSTALL_SYSTEM_RUNTIME_COMPONENT Libraries)
```

Comment:
The runtime libs should actually not be installed into the program directory but into the system directory (if newer).

Try setting `set(CMAKE_INSTALL_UCRT_LIBRARIES TRUE)`? This is for VS 2015 and newer. 

Setting CMAKE_INSTALL_SYSTEM_RUNTIME_DESTINATION should then not be required at all.

Reply:
Are you sure? 
```
CMAKE_INSTALL_UCRT_LIBRARIES

    Set to TRUE to install the Windows Universal CRT libraries for app-local deployment (e.g. to Windows XP). This is meaningful only with MSVC from Visual Studio 2015 or higher.
```

We need to set the
CMAKE_INSTALL_SYSTEM_RUNTIME_DESTINATION
because we do not use the CMake default install location C:\Programme\mixxx\bin for the mixxx.exe as well.

Reply:
We want to use the Windows Universal CRT libraries, because we are using VS 2015 or higher. No need to install custom CRT libs.

Reply:
I have triggered a build and will check the results.

Reply:
But that are additional libraries: 
https://github.com/Kitware/CMake/blob/54dbc9c8f7c57192b2e2e7a9aa48cead672d898d/Modules/InstallRequiredSystemLibraries.cmake#L348

I don't think we need to care about unless we want to support Windows XP: 
https://docs.microsoft.com/de-de/cpp/windows/universal-crt-deployment?view=msvc-160

Reply:
Didn't work as expected.